content/blog/understanding-hpe-greenlake-identity-access-management.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Understanding HPE GreenLake Central Identity & Access Management
+date: 2022-04-19T17:43:23.594Z
+author: John Lenihan
+authorimage: /img/mugshot.png
+---
+### IAM Definitions
```

Comment:
Before starting with IAM definitions, I would highly suggest to add a short paragraph in introduction of the post to clearly state upfront what you are going to tell to the reader and why (the problem statement, what type of problem do you address). Just to grab the reader attention. Something that helps the reader understand why they want to read further.

And finish off the post with a conclusion (the key takeaways) that summarizes what it is you covered, helping the readers remember a few key items and adding in some sort of call to action.
In the Introduction, you started off by telling your readers what you were going to tell them. Then you went into details about what you wanted to tell them. Now, summarize what you told them. Focus on the salient points you want to remember. Shorten them. Reiterate them. Make them memorable. Feel free to use bullets. Most post summaries are only 1-2 paragraphs long.
We also suggest to include a Call to Action in the Summary. It could be simple, like “Please check out the HPE DEV blog for more articles on this topic.” You could also offer the reader the opportunity to connect with you directly.

Reply:
OK I added introduction and summary sections and also included a 1-liner about connecting directly.